src/util/configDir.js

```diff
@@ -0,0 +1,5 @@
+const os = require("os");
+const homedir = os.homedir();
+const configDir = ".maestro";
+
+module.exports = `${homedir}/${configDir}`;
```

Comment:
I like this. There's at least one or two other places that we can use this utility after merging this PR. I've made a note of it and will make those changes at some point.